README.md

```diff
@@ -2375,6 +2418,65 @@ DESCRIPTION
   Get a MySQLDatabase.
 ```
 
+## `mw database mysql import DATABASE-ID`
+
+Imports a dump of a MySQL database
+
+```
+USAGE
+  $ mw database mysql import DATABASE-ID -i <value> [-q] [-p <value>] [--temporary-user] [--ssh-user <value>] [--gzip]
```

Comment:
Support for a [mysql options file](https://dev.mysql.com/doc/refman/8.0/en/option-files.html) would be awesome

`--defaults-extra-file="~/files/.my.cnf"`

This is only a feature idea and could also be a improvement for a follow up pull request.

Reply:
Created #393 as a follow-up issue.